bindings/java/src/test/java/org/apache/opendal/test/behavior/FsBehaviorTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.opendal.test.behavior;
+
+import java.io.File;
+import java.util.Collections;
+import java.util.Map;
+import org.assertj.core.util.Files;
+
+class FsBehaviorTest extends AbstractBehaviorTest {
```

Comment:
We can remove those files after OPENDAL_TEST introduced?

Reply:
No. These two tests are not covered by services test so I'd prefer to test them in the default workflow. Also it's good for locally debugging.

It's not "for every service", but for memory and fs only, no more.

Reply:
Understood. So, can we delete them once they've been incorporated into the service test?

Reply:
Yes of couse. I think I should have an idea now.